src/sentry/receivers/features.py

```diff
@@ -146,11 +145,22 @@ def record_issue_assigned(project, group, user, **kwargs):
     )
 
 
-@issue_resolved_in_release.connect(weak=False)
-def record_issue_resolved_in_release(project, group, user, resolution_type, **kwargs):
-    FeatureAdoption.objects.record(
-        organization_id=project.organization_id, feature_slug="resolved_in_release", complete=True
-    )
+@issue_resolved.connect(weak=False)
+def record_issue_resolved(organization_id, project, group, user, resolution_type, **kwargs):
+    """ There are three main types of ways to resolve issues
+        1) via a release (current release, next release, or other)
+        2) via commit (in the UI with the commit hash (marked as "in_commit")
+            or tagging the issue in a commit (marked as "with_commit"))
+        3) now
```

Comment:
nit: `now` is the field name, let's make this more descriptive and say "resolving immediately via UI" or something